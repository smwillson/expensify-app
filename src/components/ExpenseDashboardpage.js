import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import React from 'react';


const ExpenseDashboardpage = () => (
    <div>
        <ExpenseListFilters />
        <ExpenseList />
    </div>
);



export default ExpenseDashboardpage;