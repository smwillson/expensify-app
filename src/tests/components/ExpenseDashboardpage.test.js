import ExpenseDashboardpage from '../../components/ExpenseDashboardpage';
import React from 'react';
import { shallow } from 'enzyme';

test('Expense dashboard should be displayed to user.',()=>{
    const wrapper = shallow(<ExpenseDashboardpage/>);
    expect(wrapper).toMatchSnapshot();
   });