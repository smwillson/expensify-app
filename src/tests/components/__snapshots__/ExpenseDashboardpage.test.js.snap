// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expense dashboard should be displayed to user. 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
